themes/jquery/functions.jquery.php

```diff
@@ -177,3 +177,20 @@ function jq_logo_link() {
 	return empty( $sites[ JQUERY_LIVE_SITE ][ 'logo_link' ] ) ? '/' :
 		$sites[ JQUERY_LIVE_SITE ][ 'logo_link' ];
 }
+
+/**
+ * Meta description about current site
+ * Setup descriptions in 'sites.php'
+ *
+ * @return [String] description
+ */
+function jq_meta_description() {
+	$default = 'jQuery: The Write Less, Do More, JavaScript Library';
```

Comment:
I'd actually prefer if we default to just not having a description. Any default will be wrong for the majority of sites.